python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -146,11 +157,16 @@ def __init__(self, config_dir=None, initial_state=State.STANDBY):
                                   'u': None,
                                   'v': None
                                   }
-        self.focus_offset_per_category = {'total': 0.0,
-                                          'userApplied': 0.0,
-                                          'filter': 0.0,
-                                          'disperser': 0.0,
-                                          'wavelength': 0.0}
+
+        self.focusOffsetPerCategory = {'total': 0.0,
+                                       'userApplied': 0.0,
+                                       'filter': 0.0,
+                                       'disperser': 0.0,
+                                       'wavelength': 0.0}
+
+        self.pointingOffsetsPerCategory = {'total': np.zeros((2)),
```

Comment:
Should not be camelCase

Reply:
also changed a bunch more... but ones that don't occur as often...